.konflux/scripts/subscription-manager/subscription-manager-bro.sh

```diff
@@ -116,11 +112,6 @@ function usage {
     echo "Find out where to get it from ${SECRET_INFO_URL}"
```

Comment:
When I run `subscription-manager-bro.sh --help`, I see that `${SECRET_LOCAL_PATH}` on the line above (I can't comment there) is misleading. You can use `SCRIPT_DIR` and `SECRET_KEY` to fix that.
![image](https://github.com/user-attachments/assets/05700f9d-7d32-4236-9aaa-0c6d0b5fd477)

Reply:
done in https://github.com/stackrox/stackrox/pull/12147/commits/9d8e8112a16ba3b69ab6ed81e3e39f3429ea40c4